### PR TITLE
feat(IT Wallet): [SIW-3221] Add eID reissuance landing screen for links

### PR DESCRIPTION
**Pull request by gispada**
## Short description
This PR adds a landing screen for deep links to `credential/reissuance/eid` that allows a user to start the reissuing flow from outside the wallet.

## List of changes proposed in this pull request
- Added `ItwIssuanceEidReissuanceLandingScreen` with a temporary error message (edge cases will be handled in a future PR)
- Grouped routes that are accessible from links under `ITW_ROUTES.LANDING_SCREEN`

## How to test
Mock a message with the following cta `ioit://itw/credential/reissuance/eid` and ensure it is possible to renew the eID. When the Wallet is not active, ensure a generic error screen is displayed.

### CODE REVIEW

File: ts/features/itwallet/navigation/routes.ts [hunk 1]
```diff
@@ -61,5 +59,10 @@ export const ITW_ROUTES = {
   PLAYGROUNDS: ITW_PLAYGROUND_ROUTES,
   SETTINGS: "ITW_SETTINGS" as const,
   IDENTITY_NOT_MATCHING_SCREEN: "ITW_IDENTITY_NOT_MATCHING_SCREEN" as const,
-  WALLET_REVOCATION_SCREEN: "ITW_WALLET_REVOCATION_SCREEN" as const
+  WALLET_REVOCATION_SCREEN: "ITW_WALLET_REVOCATION_SCREEN" as const,
+  LANDING_SCREEN: {
```
Comment (mastro993) [inline ts/features/itwallet/navigation/routes.ts]: What do you think?
```suggestion
  LANDING: {
```

Comment (gispada) [reply ts/features/itwallet/navigation/routes.ts]: Addressed in f62a86e
